src/Hasher/DefaultHasher.php

```diff
@@ -15,8 +15,14 @@ public function __construct(
 
     public function getHashFor(Request $request): string
     {
-        return 'responsecache-'.md5(
-            "{$request->getHost()}-{$request->getRequestUri()}-{$request->getMethod()}/".$this->cacheProfile->useCacheNameSuffix($request)
+        if ($request->attributes->has('responsecache.cacheNameSuffix')) {
+            $cacheNameSuffix = $request->attributes->get('responsecache.cacheNameSuffix');
+        } else {
```

Comment:
Could you rewrite this code to avoid using an `else` statement?

Reply:
sure!